scripts/system/create/entityProperties/html/js/entityProperties.js

```diff
@@ -112,12 +122,12 @@ const GROUPS = [
                     lines: "Wireframe",
                 },
                 propertyID: "primitiveMode",
-            },
+            }
         ]
     },
     {
         id: "shape",
-        addToGroup: "base",
+        label: "SHAPE",        
```

Comment:
It would be nice not to have trailing spaces in the new code, here and elsewhere.